iothub/device/tests/DeviceStreamingTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NSubstitute;
+
+namespace Microsoft.Azure.Devices.Client.Test
+{
+    [TestClass]
+    [TestCategory("DeviceStreaming")]
+    [TestCategory("Unit")]
```

Comment:
These might not be unit tests. Perhaps they were just miscategorized E2E tests?

Reply:
Otherwise, I don't see how we've put real service calls in West Central US, causing this problem.

Reply:
According to the email, we should be looking for these test names:

> ModuleNotFoundTestSasl...
> DeviceClient_Amqp_TimesOutSasl_...

Reply:
me and @abhipsaMisra  are trying to figure out where these device names are coming from. We can't seem to find them in our code:
ModuleNotFoundTestSasl...
DeviceClient_Amqp_TimesOutSasl_...

Will report ASAP

Reply:
It seems like the tests that are problematic are the ones that start with E2E_DeviceStreamingTests_Sasl*, not as previously reported by the service team.